frontend/packages/dev-console/src/components/topology/shapes/CheIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { getImageForIconClass } from '@console/internal/components/catalog/catalog-item-icon';
+import { createSvgIdUrl, SVGDefs } from '@console/topology';
+
+type CheIconProps = {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+};
+
+const FILTER_ID = 'CheIconOutlineFilterId';
+
+const CheIcon: React.FC<CheIconProps> = ({ x, y, width, height }) => (
+  <>
+    <SVGDefs id={FILTER_ID}>
+      <filter id={FILTER_ID}>
+        <feOffset result="nw" in="SourceAlpha" dx="-0.5" dy="-0.5" />
+        <feOffset result="ne" in="SourceAlpha" dx="0.5" dy="-0.5" />
+        <feOffset result="se" in="SourceAlpha" dx="0.5" dy="0.5" />
+        <feOffset result="sw" in="SourceAlpha" dx="-0.5" dy="0.5" />
+        <feMerge result="blackborder">
+          <feMergeNode in="nw" />
+          <feMergeNode in="ne" />
+          <feMergeNode in="se" />
+          <feMergeNode in="sw" />
+        </feMerge>
+        <feFlood floodColor="#FFFFFF" />
+        <feComposite in2="blackborder" operator="in" result="offsetColor" />
+        <feMerge>
+          <feMergeNode in="offsetColor" />
+          <feMergeNode in="SourceGraphic" />
+        </feMerge>
+      </filter>
+    </SVGDefs>
```

Comment:
What's the purpose here? none of our other icons do this.

Reply:
It was adding white background to the che icon, but since the decorator base color is also white, i have removed it.